components/fireworks/projects/ng-fireworks/src/lib/ng-fireworks.component.ts

```diff
@@ -20,6 +22,16 @@ export class NgxFireworksComponent {
             return;
         }
 
-        fireworks(this.id, this.options);
+        this.container = fireworks(this.id, this.options);
```

Comment:
Maybe `this.container` can be assigned in the `then` of the `fireworks` call, instead of using the `then` in the `ngDestroy` method. I forgot to export the type from the library, this will be fixed in 3.2.0 of `@tsparticles/fireworks`, `any` is good for now.